packages/dd-trace/src/plugins/database.js

```diff
@@ -82,6 +82,40 @@ class DatabasePlugin extends StoragePlugin {
     }
   }
 
+  injectDbmCommand (span, command, serviceName) {
+    const mode = this.config.dbmPropagationMode
+
+    if (mode === 'disabled') {
+      return command
+    }
+
+    let dbmTraceComment = ''
+
+    const dbmService = this.getDbmServiceName(span, serviceName)
+    const servicePropagation = this.createDBMPropagationCommentService(dbmService, span)
+
+    if (mode === 'service') {
+      dbmTraceComment = `${servicePropagation}`
```

Comment:
This doesn't need to be in a template literal like this. It should suffice to just set it exactly to `servicePropagation`. If we don't know that it's always a string, you can coerce it directly using `String()` if that's what you want to do.
```suggestion
      dbmTraceComment = servicePropagation
```